client/src/app/pages/assessment-management/assessment-settings/assessment-settings-page.tsx

```diff
@@ -99,6 +100,9 @@ const AssessmentSettings: React.FC = () => {
   const [questionnaireToDelete, setQuestionnaireToDelete] =
     React.useState<Questionnaire | null>();
 
+  const [questionnaireNameToDelete, setQuestionnaireNameToDelete] =
+    React.useState<string>("");
```

Comment:
Don't we need this ```<string>``` type var here since this is a simple string value and can be inferred from usage. Looking good to me so far, though.